pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -88,7 +88,8 @@ def reduce_fx(self) -> Callable:
     @reduce_fx.setter
     def reduce_fx(self, reduce_fx: Union[str, Callable]) -> None:
         error = (
-            "Only `self.log(..., reduce_fx={min,max,mean,sum})` are currently supported."
+            "Only `self.log(..., reduce_fx={'min','max','mean','sum'})`"
+            " or `self.log(..., reduce_fx=Callable)` are currently supported."
```

Comment:
I'd rather undo this change as it makes it sound like any passing `reduce_fx` `Callable` is supported which is not the case.